data/bus-times/express.yaml

```diff
@@ -1,27 +1,26 @@
 line: Express Bus
 schedules:
   - days: [Mo, Tu, We, Th, Fr, Sa]
-    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454498, -93.161754], [44.433355, -93.186826], [44.448913, -93.171491], [44.454498, -93.161754], [44.460719, -93.155794], [44.462210, -93.183026]]
+    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454374, -93.162189], [44.438107, -93.182720], [44.432471, -93.187155], [44.435250, -93.188399], [44.441209, -93.181852], [44.449219, -93.171813], [44.454374, -93.162189], [44.460719, -93.155794], [44.462210, -93.183026]]
     stops:
-        [St. Olaf,  Carleton,  Food Co-op, Cub/Target, El Tequila, Food Co-op, Carleton,  St. Olaf]
+        [St. Olaf,  Carleton,  Food Co-op, Papa Murpheys, Cub/Target,      Arbys, Jesse James, El Tequila, Food Co-op,  Carleton,    St. Olaf]
```

Comment:
Change `Papa Murpheys` to `Papa Murphys`? `Arbys` to `Arby's`? `Jesse James` to `Jesse James Lanes`?  Also, there are some strange spaces here—not necessarily anything wrong.  @hawkrives / @drewvolz, can you take a look at this whitespace?

Reply:
Thinking Murphys -> Murphy's (https://www.papamurphys.com/)

Reply:
Yeah, agreed.  It's wrong on the bus schedule itself, which is why I'm curious, but we can always be better than that is.